### ISSUE TITLE
Improvement on Reinstall

### ISSUE THREAD

**Issue by NP-chaonay** [quo/ithc-linux#19]
DKMS-Install command cannot use duplicately (I do not know if this kind of problem also occured to non-DKMS approch)

so in order to reinstall I have to remove dkms module first

maybe we should edit the README.md

here is my initial suggestion

from 
```
Installation with DKMS
----------------------

- Install prerequisites (e.g. Debian packages: `dkms` and `linux-headers-amd64`)
- `sudo make dkms-install`
- `sudo modprobe ithc` (or reboot)
- Check dmesg for ithc messages/errors.
```
to
```
Installation with DKMS
----------------------

- Install prerequisites (e.g. Debian packages: `dkms` and `linux-headers-amd64`)
- `sudo make dkms-install` (in order to re-install, you have to run `sudo make dkms-uninstall` first (ignore any error))
- `sudo modprobe ithc` (or reboot)
- Check dmesg for ithc messages/errors.
```

**Comment by NP-chaonay** [comment]
here is error I got from re-install without DKMS uninstall comand have executed

```
dkms add `sed -n '/^PACKAGE_NAME="\(.*\)"$/s//\1/p' dkms.conf`/`sed -n '/^PACKAGE_VERSION="\(.*\)"$/s//\1/p' dkms.conf`
Error! DKMS tree already contains: ithc-0.1
You cannot add the same module/version combo more than once.
make: *** [Makefile:23: dkms-install] Error 3
```

**Comment by quo** [comment]
Thanks. Fixed in fc06cdf.
